src/chrome/content/scaffold/scaffold.js

```diff
@@ -1133,8 +1140,8 @@ var Scaffold = new function() {
 	function _updateFrames() {
 		var doc = _browser.contentDocument;
 		
-		//Show URL of active tab
-		document.querySelectorAll("textbox.browser-url").forEach(elem => elem.value = doc.location.href);
+		// Show URL of active tab
+		document.querySelectorAll("menulist.browser-url").forEach(elem => elem.value = doc.location.href);
```

Comment:
Why is this a `querySelectorAll()`, anyway? Isn't that just because we used to have more than one of the URL bar? If there's only one now, this could just be `document.getElementById('browser-url').value = …` (after moving `browser-url` to the id).